app/models/miq_worker.rb

```diff
@@ -343,11 +343,11 @@ def queue_name
   end
 
   def start_runner
-    if ENV['MIQ_SPAWN_WORKERS'] || !Process.respond_to?(:fork)
-      start_runner_via_spawn
-    else
-      start_runner_via_fork
-    end
+    spawn_worker? ? start_runner_via_spawn : start_runner_via_fork
+  end
+
+  def spawn_worker?
+    ENV['MIQ_SPAWN_WORKERS'] || !Process.respond_to?(:fork) || worker_settings[:spawn_worker]
```

Comment:
I was hoping this would give us a way to control this on fine release, I wasn't aware of a way to change the env vars per worker type on that release but if it exists then great.

Reply:
The real "why" is to spawn not fork VMware provider workers on a fine appliance to see if it solves the drb errors connecting with the broker

Reply:
@agrare Pretty sure this `SPAWN_WORKERS` isn't available on `fine`:

https://github.com/ManageIQ/manageiq/blob/fine/app/models/miq_worker.rb

So you might need to do a bit of backporting if that was the end goal.

Reply:
Yes absolutely I was planning on needing to backport some of @jrafanie 's changes to make this an option.